packages/e2e/cypress/integration/common/geo.ts

```diff
@@ -16,7 +16,7 @@ When('I clear the search results', () => {
   cy.findByRole('textbox', {
     name: /search/i,
   }).trigger('mouseenter');
-  cy.findByRole('button', { name: 'Clear' }).click();
+  cy.findByRole('button', { name: 'Clear' }).click({ force: true });
```

Comment:
Why is this necessary? Can we add a comment if this is not immediately resolvable?

Reply:
'Clear' button somehow couldn’t be found by cypress when its display property is being toggled, so added {force: true} for the click. I see it is already being used in other places so thought maybe it is an existing Cypress issue.

Reply:
Understood -- ideally, we should avoid `force: true` if possible, or explain in comment why this is necessary. For example: https://github.com/aws-amplify/amplify-ui/blob/c11de75eaf0e0476cea56d7cd2da1ffa04d1c639/packages/e2e/cypress/integration/common/shared.ts#L178-L184

Not blocking, but I think it's worth looking at for test maintainability purposes.